packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-type-form.tsx

```diff
@@ -146,194 +194,265 @@ export const PropertyTypeForm = ({
+                  setTitleValid(true);
+                } else {
+                  setTitleValid(false);
+                }
 
-              return exists ? "Property type name must be unique" : true;
-            },
-          })}
-        />
-        <TextField
-          multiline
-          inputProps={{ minRows: 1 }}
-          label={
-            <>
-              Description{" "}
-              <Tooltip
-                placement="top"
-                title="Descriptions help people understand what property types can be used for, and help make them more discoverable (allowing for reuse)."
-                PopperProps={{
-                  modifiers: [
-                    {
-                      name: "offset",
-                      options: {
-                        offset: [0, 8],
+                return exists ? "Property type name must be unique" : true;
+              },
+            })}
+          />
+          <TextField
+            multiline
+            inputProps={{ minRows: 1 }}
+            label={
+              <>
+                Description{" "}
+                <Tooltip
+                  placement="top"
+                  title="Descriptions help people understand what property types can be used for, and help make them more discoverable (allowing for reuse)."
+                  PopperProps={{
+                    modifiers: [
+                      {
+                        name: "offset",
+                        options: {
+                          offset: [0, 8],
+                        },
                       },
-                    },
-                  ],
-                }}
-              >
-                <Box
-                  sx={{
-                    order: 1,
-                    ml: 0.75,
-                    display: "flex",
-                    alignItems: "center",
+                    ],
                   }}
                 >
-                  <QuestionIcon />
-                </Box>
-              </Tooltip>
-            </>
-          }
-          required
-          placeholder="Describe this property type in one or two sentences"
-          disabled={isSubmitting}
-          success={descriptionValid}
-          error={!!descriptionError && descriptionTouched}
-          {...register("description", {
-            required: true,
-            onChange() {
-              clearErrors("description");
-              setDescriptionValid(false);
-            },
-            validate(value) {
-              setDescriptionValid(!!value);
+                  <Box
+                    sx={{
+                      order: 1,
+                      ml: 0.75,
+                      display: "flex",
+                      alignItems: "center",
+                    }}
+                  >
+                    <QuestionIcon />
+                  </Box>
+                </Tooltip>
+              </>
+            }
+            required={descriptionRequired}
```

Comment:
required is already defined in the `register`call below, do we need both?
```suggestion
```

Reply:
Possibly, although this isn't new in this PR, I'm just making required conditional. I seem to remember `register` not returning `required` in the way it should, but I may be misremembering. We don't have time to look into this right now though, especially as this isn't new.

Reply:
Yes I recall that from experience as well, now that you mention it.